flang/runtime/numeric-templates.h

```diff
@@ -88,12 +88,16 @@ struct MaxOrMinIdentity<TypeCategory::Real, 16, IS_MAXVAL,
 
 // Minimum finite representable value.
 // For floating-point types, returns minimum positive normalized value.
-template <typename T> struct MinValue {
+template <int PREC, typename T> struct MinValue {
```

Comment:
The template argument "PREC" isn't used.  Did you mean to do something with it?

Reply:
Yes, and I do below, where it allows me to define a template specialization for real(2).

Reply:
Thanks the the explanation!